tldr.py

```diff
@@ -363,9 +378,11 @@ def main():
                         help='Override the default language')
 
     parser.add_argument(
-        'command', type=str, nargs='*', help="command to lookup"
+        'command', type=str, nargs='*', help="command to lookup", metavar='command',
+        choices=get_commands() + [[]]
```

Comment:
why the + `[[]]`?

Reply:
When the user just runs `tldr` without any command, an empty list is passed in as the command. That leads to an error that says something like "[] is not a valid choice, choices: <the full list of command>" which looks horrible. This adds empty list to the list of choices and prevents the error.